frontend/justpostme/src/components/pageControl/PageControl.js

```diff
@@ -190,19 +190,34 @@ class PageControl extends React.Component<Props> {
           <Switch>
             <Route
               path={`${path}/pending`}
-              render={() => <PendingSubmissionsContainer pageId={id} />}
+              render={() => (
+                <PendingSubmissionsContainer
+                  style="display:contents; max-width:85%;"
```

Comment:
I don't like that you manually have to enter the styles, it shouldnt be like that